src/jira.ts

```diff
@@ -303,6 +313,14 @@ class Jira {
       await this.addRemoteLink(jiraUrl, url, key);
     }
 
+    // Ensure watchers from watch group are subscribed for updated
+    const jwm = new JiraWatcherManager(jiraUrl, this.token);
```

Comment:
I wonder, do you want to add these only to new issues and leave existing issues alone?

I suspect it works either way without any significant harm but wanted to mention it.

Reply:
I think I'm declaring the accounts that should be watching jira's spawned as a consequence of this bot by adding them to the list in the action. It should always be reconciling that state, even on existing issues IMO.